### PR TITLE
Delta diff MVP: add an experimental merge delta diff button

### LINKED ISSUE #5054: Delta diff MVP: An experimental merge button that leads to a coming soon modal
When in delta diff view, make the merge button experimental, send experimental feature stats and add a coming soon modal to it.

### PR DESCRIPTION
**Pull request by talSofer**
### Linked Issue

Closes #5054

---

### Background

Intelligent merge of delta lake tables is currently unsupported and we would like to understand if there is an interest around it. 
      
### New Feature

While comparing two delta lake tables using lakeFS delta diff feature show a coming soon modal and collect stats (if they are enabled) while trying to merge the changes. 

### Testing Details

Tested manually. This change applies only to the Compare tab where the Merge operation is enabled. 

## Additional info
Screen Shot 2023-02-19 at 17 07 26
Screen Shot 2023-02-19 at 17 07 16

### CODE REVIEW

File: webui/src/lib/components/modals.tsx [hunk 1]
```diff
@@ -113,46 +113,44 @@ export const ConfirmationButton: FC<ConfirmationButtonProps> = ({ msg, onConfirm
     );
 };
 
-const ModalContainer = ({children, show = false, heading, onCancel, footer=null}) => {
+const GenericModal =  ({children, show = false, heading, onCancel, footer=null}) => {
```
Comment (johnnyaug) [inline webui/src/lib/components/modals.tsx:116]:
```suggestion
const SimpleModal =  ({children, show = false, heading, onCancel, footer=null}) => {
```

**Review by johnnyaug — APPROVED** [review]

File: webui/src/lib/components/modals.tsx (hunk 1 above)
Comment (talSofer) [reply webui/src/lib/components/modals.tsx]: done

**Review by Jonathan-Rosenberg — APPROVED** [review]
Not blocking but a thing to think about- when users look at the `Merge` button they might expect to get the normal merge functionality. For instance, a user notices that there is a new delta diff functionality, then she tries it. Looks cool, but now she wants to continue and merge the branches as she used to, so she pushes the merge button and then it says that merge is unsupported?
It might be better to use something like `Merge tables` button.

**Comment by talSofer** [comment]
> Not blocking but a thing to think about- when users look at the `Merge` button they might expect to get the normal merge functionality. For instance, a user notices that there is a new delta diff functionality, then she tries it. Looks cool, but now she wants to continue and merge the branches as she used to, so she pushes the merge button and then it says that merge is unsupported? It might be better to use something like `Merge tables` button.

Nice idea! WDYT of the current version? 

Screen Shot 2023-02-21 at 9 39 34

**Comment by Jonathan-Rosenberg** [comment]
@talSofer amazing

**Pull request merged**